housekeeper/cli/add.py

```diff
@@ -43,9 +43,7 @@ def bundle_cmd(context: click.Context, bundle_name: str, json: str):
 
     validate_args(arg=bundle_name, json=json, arg_name="bundle_name")
 
-    data = {}
-    data["name"] = bundle_name
-    data["created_at"] = str(dt.datetime.now())
+    data: Dict = {"name": bundle_name, "created_at": str(dt.datetime.now())}
```

Comment:
This dictionary gets re-used below. I'd suggest adding a pydantic module for this.

Reply:
I might include this in another PR

Reply:
see #102